source/includes/mongodb-compatibility-table-node.rst

```diff
@@ -14,7 +14,17 @@
      - MongoDB 3.2
      - MongoDB 3.0
      - MongoDB 2.6
-
+   * - 4.3
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     - ✓
+     -
+     -
+     -
+     -
```

Comment:
Comment: I think the last row header is cut off on the table. All I can see is a "check mark" without a label.
![Screen Shot 2022-01-07 at 5 01 55 PM](https://user-images.githubusercontent.com/77393289/148613288-1239c3bb-6494-4dcc-afef-9d11fbd18b0e.png)

Reply:
Good catch, I think a bug in my script added an extra check mark on line 171